NebulaWorld/MultiplayerSession.cs

```diff
@@ -62,6 +72,8 @@ public MultiplayerSession(NetworkProvider networkProvider)
 
         public void Dispose()
         {
+            CanPause = true;
```

Comment:
this is unneeded as MultiplayerSession is an instance, so after it is disposed the next time it is seen will be a new instance with fields that have default values